tests/py/dynamo/conversion/test_dropout_aten.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import torch.nn as nn
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+
+from .harness import DispatchTestCase
+
+
+class TestDropOutConverter(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ((10,), 0, False),
+            ((1, 3), 0.3, False),
+            ((2, 2, 2), 0.5),
+            ((2, 2, 2, 2), 1),
+        ]
+    )
+    def test_native_dropout(self, input_shape, p, train=False):
+        class NativeDropout(nn.Module):
+            def forward(self, input):
+                return torch.ops.aten.native_dropout.default(input, p, train)
+
+        inputs = [torch.randn(input_shape)]
```

Comment:
The test cases with `inputs = [torch.randn(input_shape)] `fail both in my local tests and in the CI.

```
======================================================================
ERROR: test_native_dropout_0 (__main__.TestDropOutConverter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/hoonkyungc/miniconda3/envs/torch-trt-10/lib/python3.9/site-packages/torch/testing/_internal/common_utils.py", line 2741, in wrapper
    method(*args, **kwargs)
  File "/home/hoonkyungc/miniconda3/envs/torch-trt-10/lib/python3.9/site-packages/parameterized/parameterized.py", line 620, in standalone_func
    return func(*(a + p.args), **p.kwargs, **kw)
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/tests/py/dynamo/conversion/test_dropout_aten.py", line 24, in test_native_dropout
    self.run_test(
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/tests/py/dynamo/conversion/harness.py", line 311, in run_test
    super().run_test(
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/tests/py/dynamo/conversion/harness.py", line 72, in run_test
    interpreter_result = interpreter.run()
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py", line 323, in run
    super().run()
  File "/home/hoonkyungc/miniconda3/envs/torch-trt-10/lib/python3.9/site-packages/torch/fx/interpreter.py", line 145, in run
    self.env[node] = self.run_node(node)
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py", line 364, in run_node
    trt_node: torch.fx.Node = super().run_node(n)
  File "/home/hoonkyungc/miniconda3/envs/torch-trt-10/lib/python3.9/site-packages/torch/fx/interpreter.py", line 202, in run_node
    return getattr(self, n.op)(n.target, args, kwargs)
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py", line 472, in call_function
    return converter(self.ctx, target, args, kwargs, self._cur_node_name)
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py", line 3264, in aten_ops_native_dropout
    return impl.unary.native_dropout(
  File "/home/hoonkyungc/workspace/TorchTRT/TensorRT/py/torch_tensorrt/dynamo/conversion/impl/unary/ops.py", line 588, in native_dropout
    mask = np.ones(input_val.shape, dtype=np.bool)
  File "/home/hoonkyungc/miniconda3/envs/torch-trt-10/lib/python3.9/site-packages/numpy/__init__.py", line 324, in __getattr__
    raise AttributeError(__former_attrs__[attr])
AttributeError: module 'numpy' has no attribute 'bool'.
`np.bool` was a deprecated alias for the builtin `bool`. To avoid this error in existing code, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

```

Reply:
Thanks for the details!